cf-testd/cf-testd.c

```diff
@@ -632,8 +633,32 @@ static void *CFTestD_ServeReport(void *config_arg)
             Log(LOG_LEVEL_ERR, "Error reading report file '%s'", report_file);
             exit(EXIT_FAILURE);
         }
-        config->report     = StringWriterClose(contents);
-        config->report_len = SafeStringLength(config->report);
+
+        size_t report_data_len = StringWriterLength(contents);
+        char *report_data = StringWriterClose(contents);
+
+        Seq *report = SeqNew(64, NULL);
```

Comment:
@craigcomstock `NULL` here means that there is no destroy function, the elements are not freed when doing `SeqDestroy()`